src/DatePicker.tsx

```diff
@@ -55,13 +55,13 @@ function DatePickerWrapper(props: DatePickerWrapperProps) {
 			{...lessrest}
 			renderInput={(props) => (
```

Comment:
Yes. It's exactly how it is. Because here is props from renderInput, and other props came from DatePickerWrapper with highter pririty (required, helperText) and from Field via `input` and `meta`